extensions/src/main/java/org/pentaho/platform/plugin/services/email/EmailConfiguration.java

```diff
@@ -185,7 +279,10 @@ public String toString() {
     return "authenticate='" + authenticate + '\'' + ", debug='" + debug + '\'' + ", defaultFrom='" + defaultFrom + '\''
         + ", fromName='" + fromName + '\'' + ", smtpHost='" + smtpHost + '\'' + ", smtpPort=" + smtpPort
         + ", smtpProtocol='" + smtpProtocol + '\'' + ", smtpQuitWait=" + smtpQuitWait + ", userId='" + userId + '\''
-        + ", password='" + password + '\'' + ", useSsl=" + useSsl + ", useStartTls=" + useStartTls;
+        + ", password='" + password + '\'' + ", useSsl=" + useSsl + ", useStartTls=" + useStartTls
```

Comment:
this would be easily readable and maintainable using String.format()

`return String.format("authenticate='%s', debug='%s'...", authenicate, debug,...); `

https://www.javatpoint.com/java-string-format

Reply:
no further action. Take into account for future efforts.